src/pybullet_utils.py

```diff
@@ -0,0 +1,110 @@
+"""Generic utilities for Pybullet."""
+
+from typing import List, Sequence, Tuple
+import numpy as np
+import pybullet as p
+from predicators.src.settings import CFG
+
+Pose3D = Tuple[float, float, float]
```

Comment:
how were you able to avoid, throughout this file, mypy issues with the calls to pybullet methods?

Reply:
btw, I'm just including `Pose3D` here because I use it for clarity in the pybullet blocks env.